railib/api.py

```diff
@@ -325,6 +326,38 @@ def _parse_arrow_results(files: List[TransactionAsyncFile]):
             results.append({"relationId": file.name, "table": table})
     return results
 
+# polling with specified overhead
+# delay is the overhead % of the time the transaction has been running so far
+
+
+def poll_with_specified_overhead(
+    f,
+    overhead_rate: float = 0.1,
```

Comment:
Is 10% the standard for transactions? Seems a bit high.

Reply:
it's `10%` for julia sdk https://github.com/RelationalAI/rai-sdk-julia/blob/main/src/api.jl#L91